commit 45d3080b0ab9f9887e26a7f7abc3c726d5d648ef (refs/changes/47/4747/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-08-27T16:59:32+00:00 (1 year, 1 month ago)
    
    OS-7177 cpcgen induced build failures on old platforms
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/usr/src/lib/json_nvlist/json_nvlist.c b/usr/src/lib/json_nvlist/json_nvlist.c
index f365c981f3..efe4feed77 100644
--- a/usr/src/lib/json_nvlist/json_nvlist.c
+++ b/usr/src/lib/json_nvlist/json_nvlist.c
@@ -10,7 +10,7 @@
  */
 
 /*
- * Copyright 2015 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 #include <stdio.h>
@@ -331,7 +331,7 @@ hdlr_bareword(state_t *s)
 /* ARGSUSED */
 static int
 collect_number(state_t *s, boolean_t *isint, int32_t *result,
-    double *fresult __unused)
+    double *fresult)
 {
 	boolean_t neg = B_FALSE;
 	int t;
